lib/accesscontrol/permitted_action_on_object_query.rb

```diff
@@ -31,6 +31,30 @@ def can?(action_id, object_type, object_id)
       end
     end
 
+    # Allow permission on an ActiveRecord object.
+    # A grant is universally unique and is enforced at the database level.
+    #
+    # @param action_id [Integer] The action to grant for the object
+    # @param object_type [ActiveRecord::Base] The ActiveRecord model that receives a permission grant.
```

Comment:
Technically, we pass in a `Class` for `object_type`, right? I know I did in `can?` exactly what you're doing here by specifying `ActiveRecord::Base`, but I'm wondering if this should actually be `Class`. I'm still getting used to yardoc, so I could be wrong. What do you think?

Reply:
Ya I think you're right - I was talking with @rreinhardt9 about it yesterday and were talking about this scenario:

Consider we have a `LessonPolicy` and a `RestrictedLessonPolicy` -  Both of these policies define actions with different meaning BUT refer to the `Lesson` ActiveRecord model/object.  This means that the `object_type` we compare against should be different between the two policies? (in this case modifying what you have suggested to `Class`)

Reply:
Based on previous conversations, I expect `LessonPolicy` and `RestrictedLessonPolicy` to use the same `Lesson` permissions via Accessly, but to filter them with business logic. Especially since we use `object_type` to chain AR calls on in `list`.

Reply:
yup! business logic

The only argument I can think of to keep `[ActiveRecord::Base]` as the `object_type` is in support of the `list` operatio